### PR TITLE
Chat

### PR DESCRIPTION
**Pull request by marwuint**
there are still some syncing issues, someone please fix.

### CODE REVIEW

**Review by dstuden — COMMENTED** [review]
LGTM :)

File: app.js [hunk 1]
```diff
@@ -4,46 +4,99 @@ const app = express();
 const http = require('http').Server(app);
 const io = require('socket.io')(http);
 const branchName = require('current-git-branch')() || '?';
+const { Socket } = require('socket.io')
 
+const rooms = new Map();
```
Comment (MaticBabnik) [inline app.js:9]: Theese maps are cancer

**Review by MaticBabnik — APPROVED** [review]
whatever

File: app.js (hunk 1 above)
Comment (dstuden) [reply app.js:9]: LGTM :)
https://tenor.com/view/booty-bum-butt-minion-despicable-me-gif-7549898

**Pull request merged**